pkg/acsengine/const.go

```diff
@@ -32,7 +32,7 @@ const (
 )
 
 const (
-	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.5.3"
+	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.6.0-beta.1"
```

Comment:
Absolutely agree with all the 1.6 updates that need to happen. One thing is that this feature is only introduced in 1.6 so 1.6 is a prerequisite.